src/src/Helpers/Networking.cpp

```diff
@@ -1682,6 +1682,125 @@ int http_authenticate(const String& logIdentifier,
       }
     }
     #endif
+
+      # if FEATURE_OPENMETEO_EVENT
+
+    // Generate an event with the response of an open-meteo request.
+    // Example command:
+    // sendtohttp,api.open-meteo.com,80,"/v1/forecast?latitude=52.52&longitude=13.41&current=temperature_2m,weather_code,wind_speed_10m&daily=temperature_2m_max,temperature_2m_min&forecast_days=1"
+    // No need for an api key and it is free (daily requests are limited to 10,000 in the free version)
+    // Visit the URL (https://open-meteo.com/en/docs) and build your personal URL by selecting the location and values you want to receive.
+    // Supported variable kinds are current, hourly, daily!
+    // In rules you can grep the reply by the kind of weather variables with "On Openmeteo#<type> Do ..."
+    // e.g. "On Openmeteo#current Do ..."
+    // Note: hourly and daily results are arrays which can become very long.
+    // Best to make seperate calls. Especially for hourly results.
+
+    if ((httpCode == 200) && equals(host, F("api.open-meteo.com")))
+    {
+      // Length of the response is limited to 5000 characters
+      // ToDo: What would be right size?
+      if (uri.length() > 5000) {
+        addLog(LOG_LEVEL_ERROR, F("Response exceeds 5000 characters"));
+      }
+      else {
+        String str = http.getString();

```

Comment:
Please use `const` where possible.
At first glance, `str` is not changed anymore, so it should be `const String str`.